pkg/ytconfig/chyt.go

```diff
@@ -27,7 +27,18 @@ type ChytInitCluster struct {
 	Families       []string `yson:"families"`
 }
 
-func getStrawberryController() StrawberryController {
+type ChytConfig struct {
+	AddressResolver AddressResolver `yson:"address_resolver"`
+}
+
+func getStrawberryController(resolver *AddressResolver) StrawberryController {
+	chytConfig := ChytConfig{
+		AddressResolver: *resolver,
+	}
+	chytYsonConfig, err := marshallYsonConfig(chytConfig)
+	if err != nil {
+		chytYsonConfig = yson.RawValue("{address_resolver={enable_ipv4=%true;enable_ipv6=%true;retries=1000}}")
```

Comment:
I don't like this, why would the provided address resolver config be erroneous in the first place?
If it can be null let's handle that explicitly, otherwise there should be no errors. 

Also I believe we are losing `retries=1000`. Let's keep it for now in some way.

Reply:
Yes, I agree on silent error stuff. Let's just return error from this function.

Reply:
I think retries are ok, since they are hardcoded in fillAddressResolver (and we don't see diff in canondata)